### ISSUE TITLE
abnormal termination?

### ISSUE THREAD

**Issue by carrotking** [Murodese/pynab#185]
Running ubuntu server 13.10 64bit in a hyper-v vm.
PostgreSQL 9.3

When I run 'sudo python3 pynab.py start' it produces the errors:

pynab:scan ERROR (abnormal termination)
pynab:postproc ERROR (abnormal termination)
pynab:prebot ERROR (abnormal termination)
pynab:status ERROR (abnormal termination)
pynab:stats ERROR (abnormal termination)

I did not encounter any errors during my install and there is nothing useful in the /var/log/pynab/* files.  Would anyone care to suggest what I might do to figure out why I can't get it to run?

**Comment by brookesy2** [comment]
What does your init/supervisor/pynab.conf look like? Maybe the user doesn't exist?

**Comment by carrotking** [comment]
It shows:

user=www-data

for each program.  It is true that I do not have a user called 'www-data'.

**Comment by brookesy2** [comment]
I would guess that is the issue. Can you try running it with another user and see how it goes? Or adding www-data. Just make sure the user has access to the log file path :)

**Comment by carrotking** [comment]
Thanks, I appreciate you taking the time to reply.

Actually, there is already a 'www-data' user although I did not create it.  I did perform the install step 'sudo chown -R www-data:www-data pynab' after cloning the git repo.

I tried to login to the user www-data but it prompted for a password that I don't know/didn't set.  I forced a password change, logged in as www-data, attempted to start pynab but received the same errors.

I see in the supervisor.log file the very first line is `Supervisor running as root (no user in config file)`.  That is the only time that message appears.  The following message repeats whenever I try to start a the scan (or any other) process: 

```
INFO spawned: ' scan' with pid 1885
INFO exited: scan (exit status 1; not expected)
```

Definitely seems like a permissions issue but I'm not quite experienced enough with linux to understand how to fix it.
